sessionctx/variable/tidb_vars.go

```diff
@@ -509,4 +507,5 @@ var (
 	ExpensiveQueryTimeThreshold    uint64 = DefTiDBExpensiveQueryTimeThreshold
 	MinExpensiveQueryTimeThreshold uint64 = 10 //10s
 	CapturePlanBaseline                   = serverGlobalVariable{globalVal: "0"}
+	DefExecutorConcurrency                = runtime.NumCPU()
```

Comment:
Is it suitable in Kubernetes environment? @Yisaer

Reply:
I'm afraid not. `runtime.NumCPU()` would get the Host CPU Core nums which is not kubernetes-friendly.

Reply:
If we cannot get a quick fix for the container environment, we can leave it as an issue and try to find a way in future.